body-pix/package.json

```diff
@@ -39,7 +39,7 @@
     "rollup-plugin-uglify": "~3.0.0",
     "ts-node": "~5.0.0",
     "tslint": "~5.18.0",
-    "typescript": "~3.5.3",
+    "typescript": "~3.9.9",
```

Comment:
I bumped typescript higher to get the latest `OffscreenCanvas` types from `lib.dom`.
Types from `~3.5.3` were less specific about `getContext('2d')` return type and there were problems because of it.